models/classes/event/DeliveryExecutionFinish.php

```diff
@@ -23,19 +23,20 @@
 namespace oat\taoQtiTest\models\event;
 
 use oat\oatbox\event\Event;
+use oat\taoDelivery\model\execution\DeliveryExecution;
 
-class TestVariablesRecorded implements Event
+class DeliveryExecutionFinish implements Event
 {
-    private string $deliveryExecutionId;
+    private DeliveryExecution $deliveryExecutionId;
```

Comment:
Can you please rename the $deliveryExecutionId variable to $deliveryExecution?

Reply:
Good catch, changed